### PR TITLE
Add a new description for BitShuffle in README.md

### PR DESCRIPTION
**Pull request by maropu**
Since `snappy-java` has supported `BitShuffle`, this pr added a new description for that in `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,10 @@ snappy-java is a Java port of the snappy
 ## Features
   * Fast compression/decompression around 200~400MB/sec.
   * Less memory usage. SnappyOutputStream uses only 32KB+ in default.
-  * JNI-based implementation to achieve comparable performance to the native C++ version.  
+  * JNI-based implementation to achieve comparable performance to the native C++ version.
      * Although snappy-java uses JNI, it can be used safely with multiple class loaders (e.g. Tomcat, etc.).
   * Compression/decompression of Java primitive arrays (`float[]`, `double[]`, `int[]`, `short[]`, `long[]`, etc.)
+     * To improve the compression ratios of these arrays, you can use a fast data-rearrangement implementation ([`BitShuffle`](https://github.com/kiyo-masui/bitshuffle)) before compression
```
Comment (xerial) [inline README.md]: Link to BitShuffle should be `#bitshuffle-api`. Reference to kiyo-masui/bitshuffle should be elsewhere.

Comment (maropu) [inline README.md]: okay, I'll update

**Comment by maropu** [comment]
How about this fix?

File: README.md [hunk 2]
```diff
@@ -6,9 +6,10 @@ snappy-java is a Java port of the snappy
 ## Features
   * Fast compression/decompression around 200~400MB/sec.
   * Less memory usage. SnappyOutputStream uses only 32KB+ in default.
-  * JNI-based implementation to achieve comparable performance to the native C++ version.  
+  * JNI-based implementation to achieve comparable performance to the native C++ version.
      * Although snappy-java uses JNI, it can be used safely with multiple class loaders (e.g. Tomcat, etc.).
   * Compression/decompression of Java primitive arrays (`float[]`, `double[]`, `int[]`, `short[]`, `long[]`, etc.)
+     * To improve the compression ratios of these arrays, you can use a fast data-rearrangement implementation ([`BitShuffle`](http://static.javadoc.io/org.xerial.snappy/snappy-java/1.1.3-M1/org/xerial/snappy/BitShuffle.html)) before compression
```
Comment (xerial) [inline README.md]: We are already using sonatype repo for javadoc:  https://oss.sonatype.org/service/local/repositories/releases/archive/org/xerial/snappy/snappy-java/1.1.3-M1/snappy-java-1.1.3-M1-javadoc.jar/!/org/xerial/snappy/BitShuffle.html

**Comment by xerial** [comment]
Merged. Thanks!

**Pull request merged**